src/views/Hearing/Sidebar.js

```diff
@@ -102,17 +104,29 @@ class Sidebar extends React.Component {
               <div className="sidebar-section contents">
                 <h4><FormattedMessage id="table-of-content"/></h4>
                 <ListGroup>
-                  <ListGroupItem href="#hearing">
+                  {!isQuestionView && <ListGroupItem className={currentlyViewed === '#hearing' && 'active'} href="#hearing">
```

Comment:
Let's display the whole Hearing contents (and all subsections) in the sidebar at all times, instead of changing the sidebar when we go to the subsections.

That is, we should have the subsections listed in the sidebar below the Vaihtoehdot header (even if we are looking at the hearing main page). Similarly, the whole hearing contents should be visible at all times (even if we are looking at a subsection), with the current subsection highlighted.